### PR TITLE
[SPARK-31792][SS][DOCS] Introduce the structured streaming UI in the Web UI doc

### PR DESCRIPTION
**Pull request by xccui**
### What changes were proposed in this pull request?
This PR adds the structured streaming UI introduction to the Web UI doc.

image

### Why are the changes needed?
The structured streaming web UI introduced before was missing from the Web UI documentation.

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
N.A.

### CODE REVIEW

**Review by HeartSaVioR — APPROVED** [review]
LGTM

**Comment by xccui** [comment]
@gatorsmile, please take a look when it's convenient for you.

**Comment by HyukjinKwon** [comment]
ok to test

**Comment by HyukjinKwon** [comment]
cc @xuanyuanking FYI

File: docs/web-ui.md [hunk 1]
```diff
@@ -407,6 +407,20 @@ Here is the list of SQL metrics:
...
+When running Structured Streaming jobs in micro-batch mode, a Structured Streaming tab will be
+available on the Web UI. You can check the statistics of a running query or a completed query 
+via clicking a run id on the Streaming Query overview page.
+
+<p style="text-align: center;">
+  <img src="img/webui-structured-streaming-detail.png" title="Structured Streaming Query Statistics" alt="Structured Streaming Query Statistics">
+</p>
+
+The statistics page displays some useful metrics (e.g., how many records received per second and 
+the time used for different stages of a micro-batch execution) for insight into the status of your
```
Comment (xuanyuanking) [inline docs/web-ui.md:420]: How about adding a section for the metrics in operation duration histogram?

File: docs/web-ui.md [hunk 2]
```diff
@@ -407,6 +407,20 @@ Here is the list of SQL metrics:
 
 </table>
 
+## Structured Streaming Tab
+When running Structured Streaming jobs in micro-batch mode, a Structured Streaming tab will be
+available on the Web UI. You can check the statistics of a running query or a completed query 
+via clicking a run id on the Streaming Query overview page.
```
Comment (xuanyuanking) [inline docs/web-ui.md:413]: We can also check the latest exception for the failed query.

**Review by xuanyuanking — COMMENTED** [review]
Thanks Xingcan for the document! Also cc @uncleGen

File: docs/web-ui.md (hunk 1 above)
Comment (uncleGen) [reply docs/web-ui.md:420]: +1, we can also give some information for each metrics, like:
- Input Rate: The aggregate (across all sources) rate of data arriving.
- Process Rate: The aggregate (across all sources) rate at which Spark is processing data.
- Input Rows: The aggregate (across all sources) number of records processed in a trigger.
- Batch Duration: The process duration of each batch.

Comment (xccui) [reply docs/web-ui.md:420]: Sounds good. I'll add a list of metrics.

File: docs/web-ui.md (hunk 2 above)
Comment (xccui) [reply docs/web-ui.md:413]: Oh, sorry for missing this part. I'll add this.

**Review by xuanyuanking — APPROVED** [review]
LGTM, thanks Xingcan for the changes! cc @HyukjinKwon @gatorsmile 
Let's change the title to "[SPARK-31792][SS][DOCS] ..."

**Review by HyukjinKwon — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
Merged to master and branch-3.0.

**Pull request closed without merging**